functional_test.go

```diff
@@ -119,7 +119,7 @@ func prepareDockerTestEnvironment(ctx context.Context, env *testEnvironment) err
 	// found here: https://docs.confluent.io/current/installation/versions-interoperability.html
 	var confluentPlatformVersion string
 	switch env.KafkaVersion {
```

Comment:
Need to remember to bump `confluentPlatformVersion` to 5.6 once images from that are available I guess. Probably worth opening a backlog issue so we don’t forget. 

I did also wonder if we should move to [strimzi/kafka](https://hub.docker.com/r/strimzi/kafka) for our CI image

Reply:
I see that we use a mix of confluent-oss and [wurstmeister/kafka](https://hub.docker.com/r/wurstmeister/kafka/) internally on CI. /cc @d1egoaz

Reply:
either should work, 🤷 ideally, we should only use one flavour